example/stochastic_subgrid/stochastic_subgrid_driver.cpp

```diff
@@ -17,7 +17,7 @@
 
 // Local Includes
 #include "amr_criteria/refinement_package.hpp"
-#include "bvals/cc/bvals_cc_in_one.hpp"
+#include "bvals/bnd_flx_communication/bvals_in_one.hpp"
```

Comment:
Just thinking out loud here... does `bnd_flx_communication` need to be this verbose/does this provide necessary deconfliction with other (potential) directories in `bvals` or could we just have something like `bvals/comms/bvals_in_one.hpp`? At any rate the `flx` without `var` here makes me think this folder is specifically for flux corrections. 

Or if we need to separate variable and swarm communication into separate folders, maybe `bvals/var_comms` and then later also `bvals/swarm_comms`?

I'm not demanding any names be changed here I'm just bringing this up and it's fine to ignore me on this.

Reply:
I like `comms` better, there was no real reason for the verbosity except that I couldn't think of anything better when I wrote it. I will make the switch.